README.md

```diff
@@ -96,7 +96,7 @@ await mongoose.connect('mongodb://127.0.0.1/my_database');
 
 Once connected, the `open` event is fired on the `Connection` instance. If you're using `mongoose.connect`, the `Connection` is `mongoose.connection`. Otherwise, `mongoose.createConnection` return value is a `Connection`.
 
-**Note:** _If the local connection fails then try using 127.0.0.1 instead of 127.0.0.1. Sometimes issues may arise when the local hostname has been changed._
+**Note:** _If the local connection fails then try using 0.0.0.0 instead of 127.0.0.1. Sometimes issues may arise when the local hostname has been changed._
```

Comment:
Thanks @ankushknr19 
This is a typo and the current state of things does not make sense, that being said, the original change was the suggestion below.
```suggestion
**Note:** _If the local connection fails then try using 127.0.0.1 instead of localhost. Sometimes issues may arise when the local hostname has been changed._
```